libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCredentials.m

```diff
@@ -30,9 +30,10 @@
 static NSString * const kSFOAuthAccessGroup            = @"com.salesforce.oauth";
 static NSString * const kSFOAuthProtocolHttps          = @"https";
 
-NSString * const kSFOAuthServiceAccess          = @"com.salesforce.oauth.access";
-NSString * const kSFOAuthServiceRefresh         = @"com.salesforce.oauth.refresh";
-NSString * const kSFOAuthServiceActivation      = @"com.salesforce.oauth.activation";
```

Comment:
I couldn't find anywhere this one was used

Reply:
It can be removed. That flow has not been supported for a long time now.